src/services/localstorage.service.js

```diff
@@ -0,0 +1,25 @@
+export const clearLocalStorage = () => {
+  const horondi = {
+    accessToken: null,
+    _id: null,
+    wishlist: null,
+    cart: null,
+    language: 0
+  };
+  localStorage.setItem('horondi', JSON.stringify(horondi));
+};
+
+if (!localStorage.getItem('horondi')) {
```

Comment:
Please, put this code inside `index.js` or `app.js` to ensure that it is easy to find a place where resetting is performed

Reply:
moved code to index.js